cmd/derpprobe/derpprobe.go

```diff
@@ -93,6 +103,27 @@ func getOverallStatus() (o overallStatus) {
 			}
 		}
 	}
+
+	var subjs []string
+	for k := range certs {
+		subjs = append(subjs, k)
+	}
+	sort.Strings(subjs)
+	reissueTime := time.Unix(1643226768, 0)     // assume certs before this need reissuance by LetsEncrypt due to ALPN bug
```

Comment:
Maybe make this a const with a small description of the let's encrypt issue

Reply:
Done